### ISSUE TITLE
New feature: Level of confiment measures per geographic areas

### ISSUE THREAD

**Issue by Hytkiller** [CSSEGISandData/COVID-19#1228]
It would be interesting to include a measure on the evolution of restriction measures over time applied per contry or geographic areas on a scale such as "High, Medium, Low".
It would help in comparing the potential effectiveness of measures against the evolution of the contagion rate, death rate, recovery rates and other KPI.
The assessment metholodgy of the restriction measures is not clear to me yet though.
Any thoughts on the feature & the methodology?

**Comment by JiPiBi** [comment]
I agree, so few variables are available to understand the evolutions  . 

Knowing the number of people in ICU and the global capacity to cope with the growing values should be interesting also , but at a local level , like in Lombardia (but perhaps that are confidential data)

**Comment by yetzt** [comment]
It will be tough to agree on critarea for a comparable scale, since concrete measures vary vastly, even between areas in countries.

**Comment by JiPiBi** [comment]
@yetzt 
I agree it must be made at a regional level to be interesting 
I dont know if a site already tried to create this features , I fear that it could only be made by cooperation with many people , using a common scale and  apart from here where evolutions will be very limited (I dont know if those data a fork could make the job)

**Comment by Canoming** [comment]
I guess a better idea is to include links to the official announcements about the restrictions. But still, it can dramatically increase the workloads. Adding the "level of restriction" is not very practical.

One problem is that, the "measure of restriction" is not an objective measure. Too many factors are involved in any policies. Also, an "announced" restriction is different from the "actual" restriction, especially when many countries ordered lockdown without specifying the punishments. Not saying the cultural and political background may play roles in the judgments.

I hope this repo can keep focusing on scientific data instead of endless arguments. Even there is the ISO 3166 standard of the country codes, there are still tons of issues complaining about the country names. I don't think it's a good idea to include more controversial subjects to this dataset.
